interceptors/ratelimit/ratelimit.go

```diff
@@ -9,17 +9,17 @@ import (
 )
 
 // Limiter defines the interface to perform request rate limiting.
-// If Limit function return true, the request will be rejected.
+// If Limit function returns an error, the request will be rejected with status code and the provided error as the response.
```

Comment:
```suggestion
// If Limit function returns an error, the request will be rejected with the gRPC codes.ResourceExhausted and the provided error.
```
I mean this (: Sorry for being not clear.